libraries/tracking/streams/pages.js

```diff
@@ -12,11 +11,10 @@ import { isPWAVisible } from '../helpers';
  * @type {Array}
  */
 export const blacklistedPaths = [
```

Comment:
Please rename this to `blacklistedPatterns` and extend its values to match the route patterns.